rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/CostAccountingSpec.scala

```diff
@@ -89,6 +105,31 @@ class CostAccountingSpec extends FlatSpec with Matchers with PropertyChecks with
     }
   }
 
+  it should "be repeatable" in
+    forAll(contracts) { (contract: String, _) =>
+      checkRepeatableCost {
+        val result = evaluateWithCostLog(Integer.MAX_VALUE, contract)
+        assert(result._1.errors.isEmpty)
+        result
+      }
+    }
+
+  def checkRepeatableCost(block: => (EvaluateResult, Chain[Cost])): Unit = {
+    val repetitions = 20
+    val first       = block
+    // execute in parallel to trigger different interleaves
+    val subsequents = (1 to repetitions).par.map(_ => block).toList
+    // check assertions sequentially to avoid "suppressed exceptions" output on assertion failure
+    subsequents.foreach { subsequent =>
+      val expected = subsequent._1.cost.value
```

Comment:
Nit: I'd reverse the names `expected` and `actual`.

Reply:
So would I :D

Reply:
Done